client/src/components/molecules/style/RequestsTable.scss

```diff
@@ -0,0 +1,45 @@
+.request-list {
+    padding: 0;
+    h1 {
+        @include section-header-text;
+        padding-bottom: 4px;
+        margin: 0;
+    }
+    min-width: 1000px;
+    width: 1185px;
+}
+
+.request-table {
+    font-family: "Inter";
+    margin-top: 20px;
+    width:1200px;
+    margin-left: auto;
+    margin-right:auto;
+    .edit{
+        color: #333333;
+    }
+    .delete{
+        color: #333333;
+    }
+    & td {
+      border-top: unset;
+    }
+    .row-style{
+        font-size: 16px;
+        text-align: center;
+    }
+    .header-style{
+        font-size: 14px;
+        text-align: center;
+        color: #969696;
+    }
+  }
+
+  .btn-cont{
```

Comment:
This is another common class name. Nest it inside the .request-list selector to avoid modifying other components